docs/network/fees.md

```diff
@@ -27,23 +27,23 @@ If the transaction's base fee is not enough to cover Etherlink's fees, the trans
 
 The execution fee changes based on the transaction throughput over time.
 
-Etherlink measures throughput in terms of _ticks_, which are execution steps.
-As demand increases, Etherlink runs more ticks per second to process more transactions in the same amount of time.
+Etherlink measures throughput in terms of the amount of execution gas (that is, the units of computation used to execute transactions) used over time.
+As demand increases and more execution gas is used in a shorter time, Etherlink increases the gas price to increase the execution fee.
 
-Etherlink has a target number of ticks to run per second, referred to as the _speed limit_.
-To calculate the throughput, Etherlink stores the number of ticks used by transactions and subtracts the speed limit every second.
-The remaining number of ticks is known as the _backlog_.
+Etherlink has a target amount of execution gas to use per second, referred to as the _speed limit_.
```

Comment:
Maybe we can mention it's 2 million gas/s?

Reply:
Would that number be meaningful to Etherlink developers?

Reply:
Maybe, but it's a good data point to compare with other chains.

Reply:
Ok, added "The speed limit is currently set to 2 million gas per second."